src/main/kotlin/racingcar/domain/Cars.kt

```diff
@@ -0,0 +1,21 @@
+package racingcar.domain
+
+import racingcar.constant.ERROR_CARS_CREATE
+
+class Cars(names: List<String>) {
```

Comment:
```suggestion
class Cars(names: List<RacingCar>) {
```
클래스명만 봤을 때에는 "여러 대의 자동차"를 의미하지만, 생성자로 캡슐화한 값은 "자동차의 이름"입니다.
클래스가 캡슐화하고자 하는 프로퍼티를 캡슐화해보면 어떨까요?